src/library/Dropdown/Dropdown.js

```diff
@@ -1,21 +1,22 @@
 /* @flow */
-import React, { Component } from 'react';
+import React, { Children, Component, cloneElement } from 'react';
```

Comment:
Not sure if this is correct, but I've been ordering imports roughly like so (each group is alpha-sorted by imported filename):

1. Node packages with 'react' in the filename
2. Other node packages
3. Lowercase directories (styles, themes, utils, etc...)
4. Components
5. Subcomponents or other files in the same directory

Reply:
If we do decide on a convention, I'll just let this comment serve that purpose for all files in this PR.

Reply:
Popover.js is a good example.